### PR TITLE
add lint check workflow

### LINKED ISSUE #4: create editor config
create editor config to define formating rules.

### PR DESCRIPTION
**Pull request by okuzawats**
close #4 #6

### CODE REVIEW

**Comment by okuzawats** [comment]
OK danger worked successfully

**Comment by okuzawats** [comment]
all green🎉

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Lint
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+  push:
+    branches:
+      - main
```
Comment (nanaten) [inline .github/workflows/lint.yaml:7]: Is it okay to just use the main branch?

Comment (okuzawats) [reply .github/workflows/lint.yaml:7]: I think it's enough for now. when we'll create develop branch or else we should add other branches :)

**Review by nanaten — APPROVED** [review]
👍

**Comment by okuzawats** [comment]
👍

**Pull request merged**